Milestone 1

MrSayyod opened the pull request:
- Used `create-react-app` to initialize a new React app.
- Added `prop-types` using `npm` for typechecking props.
- Added `big.js` using `npm`.
- Created a Heroku app.
- Deployed to Heroku.
- Created a dedicated repository for the project.
- Set up the code linters.
- Set the development branch as a default branch in the repository.
- Committed changes often.

MrSayyod committed:
Install prop-types

MrSayyod committed:
Install big.js

MrSayyod committed:
Add linter files

MrSayyod committed:
Install linters

MrSayyod committed:
Install stylelint

whiz25 reviewed (changes requested):
## changes required 🙏🏿 🙏🏿 🙏🏿 
Nice work setting up the project. Well done.
### changes 🔕 🔕 🔕 
- update the readme. Add a description of the app, sections for getting started, installing packages, running locally, author, contributing, etc. These sections make your repo more professional
- delete unnecessary files such as the logo.svg
- deploy the app and add a live link to the readme file

MrSayyod commented:
- Updated the readme.
- Added a description of the app, sections for getting started, installing packages, running locally, author, contributing, etc.
- Added a live link to the readme file
- Deleted unnecessary logo.svg
- Deployed the app

MrSayyod committed:
Update README

MrSayyod committed:
Fix eslinter errors

MrSayyod committed:
Fix stylelinter errors

george-shammar commented on src/reportWebVitals.js:
```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
       getCLS(onPerfEntry);
       getFID(onPerfEntry);
       getFCP(onPerfEntry);
```
- [ ] Thanks for making some the required changes. You still have multiple unneccessary files that are not related to the calculator app. e,g `logo192.png`, `logo152.png`, `favicon`, `reportWebVitals.js` e.t.c. Please remove any file that would not be directly of any use in the project.

george-shammar reviewed (changes requested):
Hi @MrSayyod , 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

MrSayyod commented:
- Deleted unnecessary files

MrSayyod committed:
Deleted unnecessary files

whiz25 reviewed (changes requested):
## changes required 🙏🏿 🙏🏿 🙏🏿 
Nice work making the requested changes. All unnecessary files have been removed. Well done!
### changes 🔕 🔕 🔕 
- fix linter errors

### Martin Afani
#### Slack: whiz25

MrSayyod committed:
Fix eslinter errors

rindrajosia reviewed (changes requested):
Hi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

 - [x] Your app should run correctly with `npm start`:
![1](https://user-images.githubusercontent.com/58552748/131453491-1e3106d0-91e5-4f47-bfd8-7ec138996d05.gif)


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

MrSayyod commented:
- Deleted unnecessary files. Now `npm start` works correctly.

MrSayyod committed:
Delete unnecessary logo

rukundoeric reviewed (approved):
Hi @MrSayyod,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
